<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>640</y>
      <width>441</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 1px solid gray;
	border-radius: 9px;
	margin-top: 0.5em;
}

QGroupBox::title{
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Adjust gamma</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="gamma_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ratio of beta_max over beta&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSlider" name="gamma_slider">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>301</width>
       <height>29</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>3.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>640</y>
      <width>151</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 1px solid gray;
	border-radius: 9px;
	margin-top: 0.5em;
}

QGroupBox::title{
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Adjust Nbeta</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nbeta</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nbeta_box">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Number of Beta to Annealing"/>
     </property>
     <property name="placeholderText">
      <string extracomment="Enter Number: 1000"/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>640</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 1px solid gray;
	border-radius: 9px;
	margin-top: 0.5em;
}

QGroupBox::title{
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Auto-run</string>
    </property>
    <widget class="QRadioButton" name="manual_run_check">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="auto_run_check">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restore_trigger">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 1px solid gray;
	border-radius: 9px;
	margin-top: 0.5em;
}

QGroupBox::title{
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Image Preprocess</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Noise Level</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="noise_level_box">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="image_path">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>24</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="path_display">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add_noise_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Noise</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>571</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 1px solid gray;
	border-radius: 9px;
	margin-top: 0.5em;
}

QGroupBox::title{
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Noisy Image</string>
    </property>
    <widget class="QLabel" name="noisy_image">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>521</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Noise Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="original_noise_level">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>590</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>570</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 1px solid gray;
	border-radius: 9px;
	margin-top: 0.5em;
}

QGroupBox::title{
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Restored Image using Ising Model</string>
    </property>
    <widget class="QLabel" name="restore_image">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>531</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Noise Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="restore_noise_level">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>590</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1202</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImage_Restoration">
    <property name="title">
     <string>Image Restoration using Ising Model</string>
    </property>
   </widget>
   <addaction name="menuImage_Restoration"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
